<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationBodyMeshDialog</class>
 <widget class="QDialog" name="RotationBodyMeshDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вращение плоского профиля</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="labelAxe">
       <property name="text">
        <string>Ось:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axeComboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Ox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oy</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="labelRaius">
       <property name="text">
        <string>Радиус:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="radius">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>32000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLayersCount">
       <property name="text">
        <string>Количество слоев элементов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="layersCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="isClosedBody">
     <property name="text">
      <string>Замкнутое тело</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelRotationAngle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Угол поворота (°):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rotationAngle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-359.000999999999976</double>
       </property>
       <property name="maximum">
        <double>359.000999999999976</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RotationBodyMeshDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RotationBodyMeshDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isClosedBody</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelRotationAngle</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isClosedBody</sender>
   <signal>toggled(bool)</signal>
   <receiver>rotationAngle</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
